docs/src/configuring-zed.md

```diff
@@ -2128,6 +2128,16 @@ Float values between `0.0` and `0.9`, where:
 }
 ```
 
+## UI Font Size
+
+- Description: The default font size for text in the UI.
+- Setting: `ui_font_size`
+- Default: `16`
+
+**Options**
+
+`integer` values
```

Comment:
This is not complete, as there are limits: https://github.com/zed-industries/zed/pull/17829
Let's mention these also?